styles/app.scss

```diff
@@ -1,6 +1,72 @@
-// @import 'yoga';
-// @import 'normalize';
+@import 'normalize';
 
-// $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
-// @import 'font-awesome';
+$fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
+@import 'font-awesome';
 @import 'box-sizing';
+@import 'container';
+
+
+$project-white: #fff;
+$projectgray: #808080;
+$projectlightgray: #d3d3d3;
+$projectgreen: #71C671;
+$projectblue: #009ACD;
+$projectfontsize: 2rem;
+
+
+// .container {
+//   height: 30rem;
+//   width: 20rem;
+//   margin: 2rem auto;
+//
+//
+//   border: 0.15rem solid $projectlightgray;
+//   border-radius: 0.5rem;
+//   padding: 2rem;
+// }
+
+.container i {
+  color: $projectlightgray;
+  font-size: 2rem;
+  display: inline-block;
+  position: relative;
+  top: -1rem;
+}
+
+.lesson {
+  color: $projectgreen;
+}
+
+.CSS {
+  color: $projectgray;
+  text-transform: uppercase;
+}
+
+.lesson, .CSS {
+  font-size: $projectfontsize;
+  font-weight: bold;
+}
+
+.lessonlist {
+  text-decoration: underline;
+  list-style: square;
+  li {
+    color: $projectblue;
+  }
+}
+
+.circle {
+  position: relative;
+	width: 6.25rem;
+	height: 6.25rem;
+	background: $projectlightgray;
+	border-radius: 3.125rem;
+  transform: translate(13.6rem, -20rem);
```

Comment:
Position `absolute` will be your friend here so that you can position the circle in reference to another element on the page instead of using such a drastic transform: translate.

Also, if you play with setting the `line-height` the same as the `height` you should be able to center the text vertically without having to have the extra `p` within the `.circle`